pkg/controller/clusters/clusterapi_test.go

```diff
@@ -94,6 +96,19 @@ func eksaCluster() *anywherev1.Cluster {
 	}
 }
 
+func TestGetCapiClusterObjectKey(t *testing.T) {
+	g := NewWithT(t)
+	eksaCluster := eksaCluster()
+
+	expected := types.NamespacedName{
+		Name:      clusterapi.ClusterName(eksaCluster),
+		Namespace: constants.EksaSystemNamespace,
+	}
```

Comment:
IMO we shouldn't use the same code being tested to generate the expected results

```suggestion
	expected := types.NamespacedName{
		Name:      "my-cluster",
		Namespace: "eksa-system",
	}
```